pkgs/development/python-modules/pywm/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib,
+pkgs,
```

Comment:
Don't bring the world inside the expression. Pick the packages you need explicitly.